src/appMain/smartDeviceLink.ini

```diff
@@ -202,6 +202,9 @@ TCPAdapterPort = 12345
 ; If the name is omitted, Core will listen on all network interfaces by binding to INADDR_ANY.
 TCPAdapterNetworkInterface =
 
+; 128 bit uuid for bluetooth device discovery. Please format as 16 seperate bytes.
+BluetoothUUID = 0x93, 0x6D, 0xA0, 0x1F, 0x9A, 0xBD, 0x4D, 0x9D, 0x80, 0xC7, 0x02, 0xAF, 0x85, 0xC8, 0x22, 0xA8
```

Comment:
Commenting this out results in a segfault

Reply:
Should include this as a default value in `profile.cc` and comment this line out by default.